### DISCUSSION TITLE
Annotation in dynamic models

### DISCUSSION THREAD

**Discussion by timjnz** [pydantic/pydantic#8400] [Question] [answered]
Hi,
I'm trying to figure out how to use [postponed annotations](https://docs.pydantic.dev/latest/concepts/postponed_annotations/) in my dynamic model. Is this just not possible?
```
from pydantic import BaseModel, create_model


class Test(BaseModel):
    a: str
    b: 'A'

create_model('A', x=(bool, ...))
Test.model_rebuild()
> PydanticUndefinedAnnotation: Unable to evaluate forward reference ForwardRef('A')

```

**Comment by Viicos** [comment, marked as answer]
You have to assign your dynamic model to some variable, in your case `A = create_model(...)`
